src/main/protobuf/build/buildfarm/v1test/buildfarm.proto

```diff
@@ -82,11 +82,18 @@ message MemoryInstanceConfig {
   int64 cas_max_size_bytes = 7;
 }
 
+enum InstanceHashFunction {
```

Comment:
So the reason this exists for the workers and the instances is that the workers have to know this and we don't yet have the capabilities API, right? (We were thinking for a while to add it, in particular the "what's my hash function" call).

Reply:
Sure! Also I had no idea that a capabilities API was in the works.